File: src/test/testdata/assetsInCode/findUsages/FindUsages3.java
import com.badlogic.gdx.graphics.g2d.TextureRegion;
import com.badlogic.gdx.scenes.scene2d.ui.Skin;
import com.gmail.blueboxware.libgdxplugin.annotations.GDXAssets;

class FindUsages3 {

  @GDXAssets(atlasFiles = "src\\findUsages/findUsages3.atlas")
  static Skin staticSkin;

  void f() {
    staticSkin.get("button-back-disabled", TextureRegion.class);
    staticSkin.getDrawable("button-back-disabled");
    staticSkin.has("button-back-disabled",  TextureRegion.class);
    staticSkin.getSprite("button-back-disabled");
  }

}

File: src/test/testdata/assetsInCode/findUsages/FindUsages3.kt
import com.badlogic.gdx.graphics.Color
import com.badlogic.gdx.scenes.scene2d.ui.Skin
import com.gmail.blueboxware.libgdxplugin.annotations.GDXAssets
import com.badlogic.gdx.scenes.scene2d.utils.Drawable

@GDXAssets(atlasFiles = arrayOf("src\\findUsages/findUsages3.atlas"))
val s: Skin = Skin()

fun f() {
  val c = s.get("button-back-disabled", Drawable::class.java)
  val d = s.getPatch("button-back-disabled")
}